news/_posts/2020-03-20-ps17-patch-release-lifecycle.md

```diff
@@ -0,0 +1,119 @@
+
+From the moment a "trigger bug" is reported, we start a 6 weeks long timer. We consider that a patch release must be delivered within these 6 weeks.
+
+### What happens in six weeks
+
+From the moment the "trigger bug" is reported, we register into the dedicated Kanban (today it means the [1.7.6. Kanban board](https://github.com/PrestaShop/PrestaShop/projects/4)) the bugs to be fixed in the patch release, whether they are trivial, minor or major.
+
+Then we start working on fixing them (or merging the bug fixes submitted by the community).
+
+Obviously, security issues are not processed the same way: when a vulnerability is reported, we explore it and fix it in a hidden manner in order to make sure hackers unaware of the vulnerability do not hear about it. We use [GitHub Security Advisories](https://help.github.com/en/github/managing-security-vulnerabilities/about-github-security-advisories) and temporary private forks to collaborate on the fix. We only publish the advisory and the fix on the day of the release[2], following [responsible disclosure](https://en.wikipedia.org/wiki/Responsible_disclosure) principle.
+
+When all of the bug fixes for the target patch version are merged, and we pressure ourselves to make it happen before the end of the 6 weeks, maintainers deliver a Release Candidate to QA team for the standard patch release test campaign. This campaign aims to find whether this patch introduces new bugs.
+
+If the campaign reports that no bugs are found, we deliver the new patch release.
+
+## Why maintain only the latest minor version ?
+
+As explained above, we maintain only one minor version.
+When PrestaShop 1.7.7.0 is out, PrestaShop 1.7.6 will be considered [End Of Life](https://en.wikipedia.org/wiki/End-of-life_(product)) and not maintained anymore. No more bug fixes for this version, no more patch releases, not even security releases[2].
+
+This project management rule might be seen as harsh, because it means that if you do not upgrade to PrestaShop 1.7.7.0, you shop will use an outdated and unmaintained version and become possibly vulnerable to newly found security issues.
+
+However this rule is actually dictated by the availability of our resources. Very often, we are working both on a patch release and the next minor release. For example 2 weeks ago we were working both on 1.7.6.4 patch release development and 1.7.7.0 Feature Freeze. The scope of PrestaShop project is very wide and outlengths current team resources, so we need to focus our efforts on the most important components.
+
+If one day our team grows, we might be able to provide patch releases on more versions, like two or three of them, or provide [Long Term Support](https://en.wikipedia.org/wiki/Long-term_support) releases. Until then we will keep the current rule.
```

Comment:
in the future, the team of maintainers should grow not only with new PrestaShop employees, but also with external people. Is it possible to turn this paragraph to introduce this idea?
Currenlty, this is still "we, PrestaShop employees, are waiting for our employer to hire new colleagues"